cli-feed-v4.json

```diff
@@ -16528,6 +16528,241 @@
           "default": "false"
         }
       ]
+    },
+    "4.47.0": {
```

Comment:
This is missing the net8 entry. We should add it before this version is published.